setup.py

```diff
@@ -1,4 +1,10 @@
 import setuptools
+from distutils.util import convert_path
+
+package_info = {}
+version_path = convert_path('pyQuARC/version.py')
+with open(version_path) as version_file:
+    exec(version_file.read(), package_info)
```

Comment:
using `exec` is generally not considered best practice. should we avoid it here?

Reply:
we're using it here because we can't just import things from inside the package in `setup.py`, can't have that dependency. And I didn't want to repeat the version in two different files. Any other way to do it? I couldn't find a better solution. @code-geek

Reply:
Best way to do this is to have the version in a txt file and read it.